medical-portal/src/API/RSBC.DMF.MedicalPortal.API/AutoMapper.cs

```diff
@@ -16,9 +25,97 @@ public MappingProfile()
                 .ForMember(dest => dest.FullName, opt => opt.MapFrom(src => src.Case.Person.FullName))
                 .ForMember(dest => dest.Birthday, opt => opt.MapFrom(src => src.Case.Person.Birthday))
                 .ForMember(dest => dest.ComplianceDate, opt => opt.MapFrom(src => src.ComplianceDate));
+
+
+            CreateMap<LegacyDocument, ViewModels.CaseDocument>()
+             .ForMember(dest => dest.SubmittalStatus, opt => opt.MapFrom(src => src.SubmittalStatus))
```

Comment:
this is fine and arguably better but just for future reference, you don't need to add mappings to properties that have the same name. Automapper will automatically map property names that match e.g. SubmittalStatus, DocumentType, Description. In most cases, it will also handle the same property name with different types e.g. DateTime SomeDate -> String SomeDate